mlflow/ml-package-versions.yml

```diff
@@ -664,8 +664,8 @@ langchain:
         pytest tests/langchain --ignore tests/langchain/test_langchain_autolog.py
       fi
   autologging:
-    minimum: "0.1.4"
-    maximum: "0.2.1"
+    minimum: "0.1.0"
```

Comment:
Practically autologging works fine with `<= 0.1.0` (validated with `0.0.244` on notebook). However, our test suite for autologging heavily depends on Runnable and new interface (`invoke`), and we need to add hundreds of skip marks to make it pass in cross version test.